msg/RadarDetection.msg

```diff
@@ -0,0 +1,11 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+# Radar detections are the discrete clusterings of points
+# Detections which are filtered into moving objects are referred to as Tracks
+
+                                            # NB: The z component of these fields is ignored for 2D tracking.
+geometry_msgs/Point position                # x, y, z coordinates of the centroid of the object being tracked.
+geometry_msgs/Vector3 velocity              # The velocity of the object in each spacial dimension.
```

Comment:
spatial

Reply:
If we use covariance to express the uncertainty, then we really need to remove 'size'. For clustering, the size of the object is a function of the distribution of the points "variance" around the center, which is just the diagonal of the covariance matrix. NB: if we go with covariance then it is easier to express the uncertainty of the velocity which is otherwise hard to name.

Whatever we decide here should be mirrored by in RadarTrack